src/components/PdfReaderComponent.js

```diff
@@ -99,28 +99,25 @@ class PdfReaderComponent {
   };
 
   /**
-   * Load and render the first page of the given pdf.
+   * Loads the pdf document, configures 'child' components, delegates
+   * page setup and rendering of the first page.
    * @param {Uint8Array} pdf data, filename or url of a PDF document
    */
-  loadPdf = (pdf) => {
-    const self = this;
```

Comment:
I didnt even think of this benefit of using async await, but we even lost “self” which is 👌